package.json

```diff
@@ -16,6 +16,7 @@
     "express": "^4.15.3",
     "github": "^9.2.0",
     "mz": "2.6.0",
```

Comment:
Explanation for using it was given [here](https://chat.zulip.org/#narrow/stream/zulipbot/subject/zuliprc.20error/near/235286).

About it's location, you're right: I'll move this to make it a devDependency. But I don't think the comment about performance you pointed out refers to adding the dependency itself, but to the fact of using it instead of the native modules (e.g. using `mz/fs` methods rather than the ones in `fs`).

Anyways, I don't think such small performance impact is an issue in this case, since it's a single-use script with an execution time < 1s. And I do think that it definitely is a great improvement in code readability.